web-app/js/portal/cart/DownloadPanelItemTemplate.js

```diff
@@ -41,25 +44,48 @@ Portal.cart.DownloadPanelItemTemplate = Ext.extend(Ext.XTemplate, {
             '    </div>',
             '    <div class="floatRight">',
             '      {[this._dataSpecificMarkup(values)]}<br>',
-            '      {[this._shareButtonMarkup(values)]}',
             '    </div>',
             '  </div>',
             '</div>'
         ];
     },
 
+    getTooltip: function(i18nId) {
+        return OpenLayers.i18n(i18nId);
+    },
+
     _getRecordTitle: function(values) {
         return values.title;
     },
 
-    _shareButtonMarkup: function(values) {
-        return String.format(
-            '      <span class="fa fa-fw fa-share-alt fa-lg"></span>' +
-            '      <input readonly onclick="this.focus(); this.select();" title="{0}" value="{1}/home?uuid={2}" />',
-            OpenLayers.i18n('shareButton'),
+    _shareButton: function(collection) {
```

Comment:
I don't think this is a great name for a function. I know it's basically the same as `_downloadButton()` below but I think function names should have a verb in them...

(It's up to you because it does match code that's already here)